src/popup/pages/AboutUs/index.js

```diff
@@ -58,15 +57,21 @@ class AboutUs extends React.Component {
       </div>
     )
   }
+  onClickPrivacyPolicy = () => {
+    openTab('https://oasisprotocol.org/privacy-policy')
+  }
   onClickGithub = () => {
-    Toast.info("Github")
+    openTab('https://github.com/oasisprotocol/oasis-wallet-ext/')
+  }
+  onClickContactUs = () => {
+    location = ('mailto:info@oasisprotocol.org')
```

Comment:
> can someone test the Contact Us link? I don't have a native email app set up

Doesn't work. Just use openTab for this too :shrug:

Reply:
ooh that did it